--- a/InforCenter/PSE/Bom/Config/MenuItem/PseBomMenu.menu
+++ b/InforCenter/PSE/Bom/Config/MenuItem/PseBomMenu.menu
@@ -346,6 +346,17 @@
       </BusinessMethod>
       <WebMethod SelectID="[TreeList_Current_MASTERID$]" PageName="ObjectMessage" IsPopup="true" IsLoopExec="false" MinWindow="false" MaxWindow="false" IsSideSlip="false" IsModal="true" />
       <WebMessageAndLogger SelectID="[TreeList_Current_MASTERID$]" TemplateLabel="MessageAndLoggerDescription.Common" WriteLogger="true" SendMessage="true" OperateType="MessageSubscribe" />
+    </WebAction>
+    <WebAction Name="TrialToMassProductionDemo" MultiMenu="false" ModuleName="PSE" LabelName="PseBomMenu.TrialToMassProductionDemo" Image="s_common_operations" ParentWebAtionName="BOMCommon" RefreshType="" RefreshedSelect="false" RefreshedHiddentObjectInspectorPage="false" DropDownItemIsHideCustom="false" IsHidden="false" IsTemplate="false" Order="2600" IsMainConfig="false">
+      <ActionChecker SelectID="[TreeList_Current_MASTERID$]" SelectMode="SINGLE" Action="" ActionType="PSEITEM" SelectType="[{&quot;ObjectType&quot;:&quot;PSEITEM&quot;,&quot;InfoList&quot;:[]},{&quot;ObjectType&quot;:&quot;PSEITEM$&quot;,&quot;InfoList&quot;:[]}]" />
+      <JSMethod Name="InforCenter_Custom_TrialToMassProduction_OpenDemo" SelectID="[TreeList_Current_MASTERID$]" IsLoopExec="false">
+        <Para Name="ItemCode" Value="[TreeList_Current_ECODE]" />
+        <Para Name="ItemName" Value="[TreeList_Current_ENAME]" />
+        <Para Name="ItemMasterID" Value="[TreeList_Current_MASTERID$]" />
+        <Para Name="ViewID" Value="[TreeList_Current_PSEITEMVIEWEID]" />
+        <Para Name="ViewType" Value="[TreeList_Current_PSEITEMVIEWVIEWTYPE]" />
+        <Para Name="TreeListID" Value="[TREELISTID]" />
+      </JSMethod>
     </WebAction>
     <WebAction Name="Refesh" MultiMenu="false" ModuleName="PSE" LabelName="Refresh" Image="p_resume" ParentWebAtionName="BomBrowse" RefreshType="RefreshCurrentAndStruct" RefreshedSelect="false" RefreshedHiddentObjectInspectorPage="false" DropDownItemIsHideCustom="false" IsHidden="false" IsTemplate="false" Order="0480" IsMainConfig="false">
       <JSMethod Name="InforCenter_PSE_Bom_Refresh" SelectID="" IsLoopExec="false">
